modules/apps/commerce/commerce-notification-service/src/main/java/com/liferay/commerce/notification/service/impl/CommerceNotificationQueueEntryLocalServiceImpl.java

```diff
@@ -253,9 +253,17 @@ public void sendCommerceNotificationQueueEntries() throws Exception {
 				FileEntry fileEntry =
 					commerceNotificationAttachment.getFileEntry();
 
-				mailMessage.addFileAttachment(
-					_file.createTempFile(fileEntry.getContentStream()),
-					fileEntry.getFileName());
+				try {
+					mailMessage.addFileAttachment(
+						new FileAttachment(
+							fileEntry.getContentStream(),
+							fileEntry.getFileName()));
+				}
+				catch (Exception exception) {
+					if (_log.isDebugEnabled()) {
+						_log.debug(exception);
+					}
+				}
```

Comment:
@abesousa do we need to surround with try catch here? I mean before the method was throwing an exception and was not being handled, let's keep the same behavior, if a file attachment is not created in the temp directory for some reason we should throw the exception.

Thank!